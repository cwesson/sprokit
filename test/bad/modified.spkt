
func test(const con : int32, const v : bool) : int32 {
	if v == true {
		con = 2;
	} else {
		v = false;
	}
	return 1;
}
