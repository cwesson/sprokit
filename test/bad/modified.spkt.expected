Error on line 4:
  Attempt to modify constant con
Error on line 6:
  Attempt to modify constant v
